crates/pessimistic-proof-core/src/aggchain_proof.rs

```diff
@@ -53,7 +52,7 @@ impl AggchainData {
                 aggchain_vkey,
             } => keccak256_combine([
                 &(ConsensusType::Generic as u32).to_be_bytes(),
-                words_to_bytes_le(aggchain_vkey).as_slice(),
+                words_to_bytes_be(aggchain_vkey).as_slice(),
```

Comment:
I don't have the crate dependency graph fresh in mind but could we somehow reuse `aggkit_prover_types::vkey_hash::VKeyHash` type which already has methods for these conversions? If not, at least add an issue for it, so we can reuse the logic.

Reply:
No we can't. Aggkit and pessimistic-proof are two different domains.
We can move this when we'll have a shared repo with everything related to interop